create-app.ts

```diff
@@ -130,8 +130,8 @@ export async function createApp({
     console.log(
       yellow(
         `You have selected tools that require configuration. Please configure them in the ${terminalLink(
-          "config/tools.json",
-          `file://${root}/config/tools.json`,
+          "config/tools.yaml",
```

Comment:
Do we also show this for typescript?

Reply:
This is in the shared code and python specific. We either have to move it to the python code or better language agnostic

Reply:
Yes, it's python specific but putting it here will help user have a notice about that.